### DISCUSSION TITLE
Are there any future plans to bring back custom subdomains (*.deta.app/*.deta.dev)?

### DISCUSSION THREAD

**Discussion by codyandersan** [deta/space-docs#54] [Ideas]
From the faqs:
> Custom domains are available on Deta Space. Custom sub-domains (sub.deta.dev) are not currently available.

Customising subdomains was available in the previous Deta Cloud. But in Space, I see it is no longer supported.

So, I want to know are there any future plans over it, as I don't own a domain and I used to rely heavily on your subdomains and customising them on my needs. 

Thanks in advance!

**Comment by sudo-subg** [comment]
Hey @codyandersan, custom subdomains with `_.deta.app` endpoints may never be possible.
But we will have custom subdomains such as `_.deta.dev` and maybe some more.

**Reply by codyandersan** [reply]
> Hey @codyandersan, custom subdomains with `_.deta.app` endpoints may never be possible.
> But we will have custom subdomains such as `_.deta.dev` and maybe some more.

Okay, thanks for your response 😊
